cmd/swagger/commands/generate/support.go

```diff
@@ -45,39 +43,26 @@ func (s *Support) Execute(args []string) error {
 		DumpData:      s.DumpData,
 		DefaultScheme: s.DefaultScheme,
 		TemplateDir:   string(s.TemplateDir),
-	}
+	}, nil
+}
 
-	if err := generator.GenerateSupport(s.Name, nil, nil, &opts); err != nil {
-		return err
-	}
+func (s *Support) generate(opts *generator.GenOpts) error {
```

Comment:
I'd like to either see this method renamed or the package level generate method renamed. I had to do quite a bit of digging to work out that the config file is still being read